AdyenDropIn/Utilities/ComponentManager.swift

```diff
@@ -61,8 +65,21 @@ internal final class ComponentManager {
                                             footer: paidFooter)
 
         // Stored section
-        let storedSection = ComponentsSection(components: storedComponents)
-
+        let storedSection: ComponentsSection
+        
+        if supportsEditingStoredPaymentMethods {
+            let allowDeleting = configuration.paymentMethodsList.allowDisablingStorePaymentMethods
```

Comment:
do deleting and disabling mean the same in this context?

Reply:
yes